application/src/main/java/be/jpendel/domain/event/EventFactory.java

```diff
@@ -4,6 +4,7 @@
 
 public class EventFactory {
     //TODO add check that event with same name and same startdate, location can not be created
+    //->check EventFactoryTest
```

Comment:
Das een verantwoordelijkheid dat ik zou overlaten aan de repository. Anders moet een factory weten welke events al allemaal bestaan, en de kans is groot dat we daar hiervoor ook de op de repo zullen dependen (ik zie niet in hoe we dat anders zouden doen)